docs/src/main/paradox/recordio.md

```diff
@@ -0,0 +1,61 @@
+# RecordIO Framing
+
+The codec parses an incoming RecordIO-formatted stream of ByteStrings into distinct frames.
+
+## Artifacts
+
+sbt
+:   @@@vars
+    ```scala
+    libraryDependencies += "com.lightbend.akka" %% "akka-stream-alpakka-simple-codecs" % "$version$"
+    ```
+    @@@
+
+Maven
+:   @@@vars
+    ```xml
+    <dependency>
+      <groupId>com.lightbend.akka</groupId>
+      <artifactId>akka-stream-alpakka-simple-codecs_$scala.binaryVersion$</artifactId>
+      <version>$version$</version>
+    </dependency>
+    ```
+    @@@
+
+Gradle
+:   @@@vars
+    ```gradle
+    dependencies {
+      compile group: "com.lightbend.akka", name: "akka-stream-alpakka-simple-codecs_$scala.binaryVersion$", version: "$version$"
+    }
+    ```
+    @@@
+
+## Usage
+
+The helper object @scaladoc[RecordIOFraming](akka.stream.alpakka.recordio.scaladsl.RecordIOFraming) provides a `Flow[ByteString, ByteString, _]` which parses out RecordIO frames.
```

Comment:
Mention the factory method `scanner`, as that is the one that provides the `Flow[...]`. Also add `$` to the end of `RecordIOFraming`.